spanner/spansql/sql.go

```diff
@@ -365,6 +365,18 @@ func (sel Select) addSQL(sb *strings.Builder) {
 
 func (sft SelectFromTable) SQL() string {
 	str := sft.Table.SQL()
+	if len(sft.Hints) > 0 {
+		str += "@{"
+		kvs := make([]string, len(sft.Hints))
+		i := 0
+		for k, v := range sft.Hints {
+			kvs[i] = fmt.Sprintf("%s=%s", k, v)
+			i += 1
```

Comment:
Can you please update to `i++`? I got an error from the linter: 

```
spanner/spansql/sql.go:374:4: should replace i += 1 with i++
```

Reply:
Thanks.

fixed at 83d63c5